druid-shell/Cargo.toml

```diff
@@ -11,6 +11,7 @@ edition = "2018"
 
 [features]
 use_gtk = ["gtk", "gtk-sys", "gio", "gdk", "gdk-sys", "glib", "glib-sys", "cairo-rs"]
+use_x11 = ["xcb", "cairo-sys-rs"]
```

Comment:
I would like to just call this feature `x11`. `use_gtk` should also be called `gtk`, I didn't understand at the time that an optional crate was automatically a feature.

(this is not to say you should change `use_gtk`, just not to treat it as an example).

Reply:
Done; changed all `use_x11` to just `x11`